pyCGM_Single/tests/test_pycgmStatic_axis.py

```diff
@@ -0,0 +1,1527 @@
+          np.array([-4, 8, -5])]),
+        # Testing that when frame is composed of lists of floats
+        ({'SACR': [-4.0, 8.0, -5.0], 'RASI': np.array([-6.0, 6.0, 3.0]), 'LASI': np.array([-7.0, -9.0, 1.0]),
+          'RPSI': [1.0, 0.0, -4.0], 'LPSI': [7.0, -2.0, 2.0]},
+         [np.array([-6.5, -1.5, 2.0]),
+          np.array([[-6.72928306, -1.61360872, 2.96670695], [-6.56593805, -2.48907071, 1.86812391], [-5.52887619, -1.59397972, 2.21928602]]),
+          np.array([-4, 8, -5])]),
+        # Testing that when frame is composed of numpy arrays of floats
+        ({'SACR': np.array([-4.0, 8.0, -5.0], dtype='float'), 'RASI': np.array([-6.0, 6.0, 3.0], dtype='float'),
+          'LASI': np.array([-7.0, -9.0, 1.0], dtype='float'), 'RPSI': np.array([1.0, 0.0, -4.0], dtype='float'),
+          'LPSI': np.array([7.0, -2.0, 2.0], dtype='float')},
+         [np.array([-6.5, -1.5, 2.0]),
+          np.array([[-6.72928306, -1.61360872, 2.96670695], [-6.56593805, -2.48907071, 1.86812391], [-5.52887619, -1.59397972, 2.21928602]]),
+          np.array([-4, 8, -5])])])
+    def test_pelvisJointCenter(self, frame, expected):
+        """
+        This test provides coverage of the pelvisJointCenter function in pycgmStatic.py, defined as pelvisJointCenter(frame)
+        This test takes 2 parameters:
+        frame: dictionary of marker lists
+        expected: the expected result from calling pelvisJointCenter on frame
+
+        This test is checking to make sure the pelvis joint center and axis are calculated correctly given the input
+        parameters. The test checks to see that the correct values in expected are updated per each input parameter added:
+            When values are added to frame['RASI'] and frame['LASI'], expected[0] and expected[1] should be updated
+            When values are added to frame['RPSI'] and frame['LPSI'], expected[2] should be updated
+            When values are added to frame['SACR'], expected[2] should be updated, and expected[1] should also be updated
+        if there are values for frame['RASI'] and frame['LASI']
+        Values produced from frame['SACR'] takes precedent over frame['RPSI'] and frame['LPSI']
+
+        If RPSI and LPSI are given, then the sacrum will be the midpoint of those two markers. If they are not given then the sacrum is already calculated / specified. 
+        The origin of the pelvis is midpoint of the RASI and LASI markers.
+        The axis of the pelvis is calculated using LASI, RASI, origin, and sacrum in the Gram-Schmidt orthogonalization procedure (ref. Kadaba 1990). 
+
+        Lastly, it checks that the resulting output is correct when frame is composed of lists of ints, numpy arrays of
+        ints, lists of floats, and numpy arrays of floats. frame['LASI'] and frame['RASI'] were kept as numpy arrays
+        every time as list would cause an error in pycgmStatic.py line 632 as lists cannot be divided by floats:
```

Comment:
putting line number explicitly is not a good idea since we may change that file and won't know it changed.